interface/ceed-elemrestriction.c

```diff
@@ -298,7 +298,7 @@ static struct CeedElemRestriction_private ceed_elemrestriction_none;
 /// @{
 
 /// Indicate that the stride is determined by the backend
-const CeedInt CEED_STRIDES_BACKEND[3] = {};
+const CeedInt CEED_STRIDES_BACKEND[3] = {0, 0, 0};
```

Comment:
I think the standard says you can do it this way:
```suggestion
const CeedInt CEED_STRIDES_BACKEND[3] = {0};
```